Get_product_data.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import json
+from bs4 import BeautifulSoup
+import os
+
+class Get_product_infor:
+    def __init__(self):
+        self.base_url = "https://www.amazon.co.jp/s?k={}&page={}"
+
+    def get_products_data(self, soup):
+        products = []
+        for product in soup.select("div[data-component-type='s-search-result']"):
+            product_data = {}
+            try:
+                product_data['Name'] = product.find('span', class_='a-size-base-plus a-color-base a-text-normal').text.strip()
+                product_data['Price'] = product.find('span', class_='a-offscreen').text.strip()
+                product_data['Link'] = "https://www.amazon.co.jp" + product.find('a', class_='a-link-normal')['href']
+                product_data['Image'] = product.find('img', class_='s-image')['src']
+                rating = product.find('span', {'class': 'a-icon-alt'})
+                if rating is not None:
+                    product_data['Rating'] = rating.text.strip().split()[0]
+                else:
+                    product_data['Rating'] = 'N/A'
+                products.append(product_data)
+            except:
+                continue
+        return products
+
+
+    def Create_file(self, keyword):
```

Comment:
Unlike golang, python does not request Capital the first letter for public methods. 
Convention:
- `_xxx` for private
- `xxx` for public